modules/tls-profiles-understanding.adoc

```diff
@@ -35,11 +35,6 @@ The `Intermediate` profile requires a minimum TLS version of 1.2.
 
 The `Modern` profile requires a minimum TLS version of 1.3.
 
-[IMPORTANT]
```

Comment:
This information is not in the Networking area, but I found it when searching the documentation for the `Modern` parameter. I'm unsure if removing here is too far-reaching?